_pages/dev/components/banner-component.md

```diff
@@ -2,62 +2,64 @@
+    component: BannerComponent,
+  },
 };
 ```
 
-You can use this component mapping to configure an alternative banner implementation. You might need to map all CMS banner component types in case multiple types are used in your project.
+You can use this component mapping to configure an alternative banner implementation. If you introduce an alternative component, you must map all the CMS banner component types to your new component implementation, similar to the configuration above.
 
-## Adaptive vs responsive images
+## Adaptive and Responsive Images
 
-The banner component supports both _responsive_ images as well as _adaptive_ images. Responsive images are simply driven by the CSS rules, which allow to reuse the same images cross different devices and screen sizes. Adaptive images can be used simultaneouly, and are driven by the images provided by the backend. The backend provides different medias by media formats. The `cx-media` components adds the different formats in a so-called `srcset`, so that the browser decides when to use which image.
+The banner component supports both responsive images and adaptive images. Responsive images are simply driven by CSS rules, which allow you to reuse the same images across different devices and screen sizes. Adaptive images can be used simultaneously, and are driven by the images provided by the back end. The back end provides different media items according to the media format. The `cx-media` component adds the different formats in a `srcset` so that the browser can decide when to use which image. Media items are rendered in the `cx-media` component. For more information, see [Media Component]({{ site.baseurl }}{% link _pages/dev/components/shared-components/media-component.md %}).
 
 ## Component Data
 
 The banner component renders the following properties:
 
-| config     | description                                                                                                         |
-| ---------- | ------------------------------------------------------------------------------------------------------------------- |
-| `urlLink`  |                                                                                                                     |
-| `external` | If set to true, the URL will not use the Angular routerLink, but an ordinary href instead.                          |
-| `media`    | The media contains different media per format, so that an optimzed media can be used for the given space and usage. |
-| `headline` | The headline is rendered as paragraph tag before the media.                                                         |
-| `content`  | The content is rendered in a paragraph tag below the media.                                                         |
+| Config     | Description                                                                                                          |
+| --- | --- |
+| `urlLink` | An optional link to an internal or external resource. |
```

Comment:
let's make it less academic: 

An optional link to an internal or external page.

Reply:
👍